setup.py

```diff
@@ -588,7 +628,11 @@ def default_requires():
     setup_spec.extras["default"] += setup_spec.extras["framework"]
     setup_spec.extras["default"] += setup_spec.extras["rag"]
     setup_spec.extras["default"] += setup_spec.extras["datasource"]
-    setup_spec.extras["default"] += setup_spec.extras["torch"]
+    cuda_version = get_cuda_version()
+    if cuda_version is not None:
+        setup_spec.extras["default"] += setup_spec.extras["torch_cuda"]
```

Comment:
There is no need to add "torch_cuda" to default here, otherwise it will be installed from https://download.pytorch.org/whl every time.

Reply:
This is to solve the issue for some Windows users, where if it's not checked for the presence of CUDA, installing directly will result in the absence of a torch_cuda environment.

Reply:
> This is to solve the issue for some Windows users, where if it's not checked for the presence of CUDA, installing directly will result in the absence of a torch_cuda environment.

Perhaps we can tell users in the documentation that if they encounter the problem that cuda cannot be installed, they can try to use pip install -e ".[torch_cuda]" to install dependencies.

Reply:
Okay, I will use the previous installation method.